site/_posts/2020-08-11-advanced-registration-form.md

```diff
@@ -0,0 +1,174 @@
+
+Let's also add a maximum price field and use the same settings. Use a key of `maxprice`; keys must be unique within the registration. It'll have a different name as well, but the other settings should be the same as the `minprice` field created first.
+
+Finally, add a geographic search area. This lets new users share where they are looking to buy. It'll be a string, but make it optional, as potential users might not have a good idea of where they're interested in looking at homes.
+
+{% include _image.liquid src="/assets/img/blogs/advanced-registration-forms/form-field-geographic-area.png" alt="Adding the geographic area field." class="img-fluid" figure=false %}
+
+There are many other available data types, including dates or booleans. There are also other, more targeted form controls, such as a textarea or select dropdowns. You can even store arrays and maps. Please consult [the API docs](/docs/v1/tech/apis/form-fields) for more information. 
+
+After saving the above additions, if you view the list of fields, you'll notice you've added three fields. They are available for use in the form you're going to build next. They can also be used for future forms as well.
+
+{% include _image.liquid src="/assets/img/blogs/advanced-registration-forms/list-of-form-fields.png" alt="The list of fields with our custom fields added." class="img-fluid" figure=false %}
+
+## Build the form
+
+Next you need to build the form. At this point you can mix and match any of the standard, predefined form fields and your recently added custom form fields. 
+
+Fields can appear in any order on the form; whatever makes the most sense for your audience works for FusionAuth. When you create a new form, you'll see a name field and a button to add steps:
+
+{% include _image.liquid src="/assets/img/blogs/advanced-registration-forms/initial-form.png" alt="The blank form, ready to be assembled." class="img-fluid" figure=false %}
+
+The only requirements for a registration form are:
+
+* You must have at least one step.
+* You must have either an email or a username field in one of your steps.
+* You must have a password field in one of your steps.
+
+To begin building the real estate application form, navigate to "Customizations" and then to "Forms". Click the green "+" button to create a new form, and name it. How about something snappy, like "Real estate application signup"? Add the first step, and then the following fields:
+
+* First name
+* Email
+* Password
+* Phone number
+
+When you're done, it should look like this: 
+
+{% include _image.liquid src="/assets/img/blogs/advanced-registration-forms/form-first-step.png" alt="Adding fields to our first step." class="img-fluid" figure=false %}
+
+Just as you can add many custom form fields, you can also add as many steps as you want. However, please think of the users who are filling out the form and don't add too many steps! 
```

Comment:
Maybe add a reference: https://www.searchenginewatch.com/2016/11/14/are-you-asking-for-too-much-information-on-web-forms/

Reply:
Ooh, good idea. I like Nielsen, will put a link to https://www.nngroup.com/articles/web-form-design/ in there.